org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "opent2t-translator-wink-thermostat",
+  "version": "1.0.0",
+  "description": "Wink Thermostat translator",
+  "author": "OpenT2T",
+  "license": "MIT",
+  "scripts": {
+    "test": "ava"
+  },
+      "ava": {
+      "files": [
+        "tests/*.js",
+        "!**/deviceconfig.js"
+      ]
+    },
+  "dependencies": {
+    "delay": "^1.3.1",
+    "opent2t": "*",
+    "opent2t-onboarding-winkhub": "*",
+    "opent2t-translator-helper-wink": "1.0.0",
+    "q": "1.4.1"
+  },
+  "devDependencies": {
+    "ava": "^0.15.2"
+  },
+  "opent2t": {
+    "interfaces": {
+      "org.OpenT2T.Sample.SuperPopular.Thermostat": {
```

Comment:
It is really a "module name" for the interface. So it can be either a .js file name or the name of a directory that contains an index.js file for the module.